CHANGELOG.md

```diff
@@ -4,6 +4,45 @@
 
 [1]: https://www.npmjs.com/package/@google-cloud/firestore?activeTab=versions
 
+## v0.20.0
+
+01-16-2019 13:14 PST
+
+### Bug Fixes
+- refactor: change timestampsInSnapshots default to true.. ([#520](https://github.com/googleapis/nodejs-firestore/pull/520))
```

Comment:
FYI- This is actually a breaking change (we're changing the default behavior for people not using timestampsInSnapshots). Should we flag it as more than just "refactor" somehow?  (sorry I was probably supposed to name my PR differently)

Reply:
Yeah - we should totally do that.  I'm not super up to date on the change.  Can you add that to the changelog at the top?  See this for how I flag breaking changes:
https://github.com/googleapis/google-auth-library-nodejs/pull/594/files

Reply:
Thanks.  I added a breaking change notice for timestampsInSnapshots, and a deprecation notice for v1beta1 gapic client.  Let me know if you want any tweaks.  And if there's anything more we should do in order to deprecate v1beta1, let me know.